README.md

```diff
@@ -21,6 +21,7 @@
 <!-- prettier-ignore-start -->
 [![GPL 3.0 License][license-badge]][license]
 [![Code of Conduct][coc-badge]][coc]
+[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/from-referrer/)
```

Comment:
this adds a little repo badge for gitpod

![testing-accessibilityworkshop-manual-a11y-testing Materials for the Manual Accessibility Testing Workshop 2022-04-16 at 14 05 47](https://user-images.githubusercontent.com/86834/163691339-c621bdcd-c07b-44ea-a8c2-45fc2b3d295d.jpg)